src/server/routes/monthData.js

```diff
@@ -90,8 +89,9 @@ function selectMonthData(req, res, next) {
   if (!seasonMatches || !isValidAudience) {
     console.error('Status Code: 400, Error Message: Invalid season or audience.');
 
-    return; res.redirect('/books-music-dvds/recommendations/404');
-  } else {
+    return res.redirect('/books-music-dvds/recommendations/404');
```

Comment:
`'/books-music-movies/recommendations/404'` here?

Reply:
This should be updated once your PR is merged to `development` and I merge from new `development` again.

Reply:
I see. :+1: